src/drivers/camera_trigger/camera_trigger.cpp

```diff
@@ -257,6 +271,15 @@ CameraTrigger::CameraTrigger() :
 	_turning_on(false),
 	_last_shoot_position(0.0f, 0.0f),
 	_valid_position(false),
+	_pseudo_oblique_num_poses(0),
```

Comment:
I realize you're being consistent with what's already in place, but would you mind doing this as in-class member initializers? It makes it easier to ensure everything is initialized and then you don't have to fight initializing ordering.

Reply:
Indeed I was following the standard in place, I wouldn't mind changing it, but I wonder what the other developers would think about having two standards in the same file.